d2common/d2data/d2datadict/skills.go

```diff
@@ -281,7 +281,7 @@ func LoadSkills(file []byte) {
 	for d.Next() {
 		record := &SkillRecord{
 			Skill:             d.String("skill"),
-			ID:                d.Number("Id"),
+			ID:                d.Number("ID"),
```

Comment:
we should not be altering the name of the column  we are retrieving from the txt files